src/assets/room_devices/pet_bowl/pet_bowl.asset.ts

```diff
@@ -3,20 +3,20 @@ DefineRoomDeviceAsset({
 	size: 'medium',
 	colorization: {
 		bowl: {
-			name: 'Pet bowl',
+			name: 'Pet Bowl',
```

Comment:
Oh, I did not mean the name of the colors, those are not so important, as they are not shown in a list of assets in the wardrobe.

I meant the asset name in line 2.

I just now looked over all assets in the wardrobe and they are all capitalized, except this one, the catsuit, the finger nails and the nylon socks. ^^"

Reply:
But we can also do those another time and simply merge it like this for now.